pages/overview/similar-technologies.md

```diff
@@ -0,0 +1,22 @@
+---
+section: docs
+title: Similar Technologies
+---
+
+While we think that JSON Schema has a unique value proposition, we'd like to provide visibility to other technologies that share similar goals:
+
+
+
+* **JSON Constraint Rules (JCR):** JCR aims to provide a more expressive way to define constraints on JSON data compared to JSON Schema. While still under development, you can learn more about it on GitHub: [http://codalogic.github.io/jcr/](http://codalogic.github.io/jcr/).
+
+* **CBOR Constrained Data Description Language (CBL):** Similar to JCR, CBL focuses on describing constraints, but specifically for the CBOR data format. Dive deeper into CBL in the CBL IETF draft: [https://datatracker.ietf.org/doc/draft-cordell-jcr-co-constraints/](https://datatracker.ietf.org/doc/draft-cordell-jcr-co-constraints/).
+
+* **CBOR Data Definition Language (Cddl):** While not strictly focused on constraints, Cddl allows defining the structure of CBOR data. Check out the Cddl IETF draft: [https://tools.ietf.org/html/draft-greevenbosch-appsawg-cbor-cddl-08](https://tools.ietf.org/html/draft-greevenbosch-appsawg-cbor-cddl-08) for more information.
+
+* **AJV (Another JSON Schema Validator):** A high-performance JSON Schema validator with extensive support for various features. Explore it at AJV website: [https://ajv.js.org/](https://ajv.js.org/).
+
+## Contributing to the Community
+
+Do you know of any other relevant technologies that we should add to this list? We encourage contributions! Please submit a pull request to the relevant contributions page on the JSON Schema website.
```

Comment:
```suggestion
Do you know of any other relevant technologies that we should add to this list? Please submit a pull request. We encourage contributions! 
```

Reply:
@Michael-Obele Last suggestion! As soon as we commit this I'll merge the PR. Nice job!

Reply:
This is an excellent suggestion, and I will implement it right away. Thanks